packages/web-react/src/components/UNSTABLE_ProductLogo/README.md

```diff
@@ -0,0 +1,41 @@
+# UNSTABLE ProductLogo
+
+⚠️ This component is UNSTABLE. It may significantly change at any point in the future. Please use it with caution.
+The UNSTABLE_ProductLogo component is used to display a product logo.
+
+The `UNSTABLE_ProductLogo` component is used to display the logo of the product.
+
+```jsx
+import { UNSTABLE_ProductLogo } from '@spirit/web-react';
+
+<UNSTABLE_ProductLogo>
+  <!-- Logo go here -->
+</UNSTABLE_ProductLogo>;
+```
+
+### Inverted color
+
+You can add an `isInverted` prop to invert the color of the logo.
+
+```jsx
+import { UNSTABLE_ProductLogo } from '@spirit/web-react';
+
+<UNSTABLE_ProductLogo isInverted>
+  <!-- Inverted logo go here -->
+</UNSTABLE_ProductLogo>;
+```
+
+## API
+
+| Name         | Type        | Default | Required | Description                        |
+| ------------ | ----------- | ------- | -------- | ---------------------------------- |
+| `children`   | `ReactNode` | `null`  | ✓        | Content of the ProductLogo         |
+| `isInverted` | `bool`      | `false` | ✕        | If true, Logo is in inverted color |
```

Comment:
?

```suggestion
| `isInverted` | `bool`      | `false` | ✕        | If true, Logo has inverted colors |
```

Because we change `color` and `background-color`.